### ISSUE TITLE
`uSecurityTlsRemove` Memory Leak

### ISSUE THREAD

**Issue by wrh-dev** [u-blox/ubxlib#70]
Thanks for creating this library!

I see a small bug that results in a memory leak in `u_security_tls.c`. I think this code:

```c
void uSecurityTlsRemove(uSecurityTlsContext_t *pContext)
{
    if ((pContext != NULL) && init()) {
...
```

Should be updated to:

```c
void uSecurityTlsRemove(uSecurityTlsContext_t *pContext)
{
    if ((pContext != NULL) && (init() == 0)) {
...
```

to account for the return type of the `init()` function.

Does this change make sense?

**Comment by RobMeades** [comment]
Hi there and thanks very much for spotting this: someone in one of our teams spotted it also, not more than a month ago, so you're very on the ball!  You'll find the fix here in this commit 2bce269d5ed03598765e672874b6d0118a871f81.

**Comment by wrh-dev** [comment]
Oh, perfect! I'll go ahead and close out this issue. Thanks!
